src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -789,6 +795,26 @@ private byte[] GetSidBinaryFromWindows(string domain, string user)
             }
         }
 
+        public bool IsTheSameUserLoggedIn(string domainName, string userName)
```

Comment:
For readability it would be good to stick with one terminology rather than jumping from "IsTheSameUserLoggedIn" to "IsActiveSessionExists", especially since this code goes through a couple layers here.

Also HasActiveSession might be more straight forward language. It is also more precise than "IsTheSameUserLoggedIn" which almost seems to imply the current login session.